### PR TITLE
feat: initialize the front end of proposition and question bank system

### PR DESCRIPTION
**Pull request by stacker-l**
Initialize the front-end framework of question bank and proposition system, use react as the project
framework, add antd, Axios, customize CRA, less, prop types, react app rewired, react Redux, react
router Dom and Redux dependencies, and add some project configuration files.

BREAKING CHANGE: empty

### CODE REVIEW

**Comment by bnuluohf** [comment]
@stacker-l demo link?

File: package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (bnuluohf) [inline package.json:1]: 最外层的packger.json没啥用

File: web-client/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (bnuluohf) [inline web-client/README.md]: web-client改成web

**Review by bnuluohf — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (stacker-l) [reply package.json:1]: > @stacker-l演示链接？

https://www.chinamfgzs.cn/open-ct

File: web-client/README.md (hunk 2 above)
Comment (stacker-l) [reply web-client/README.md]: 已修改

File: package.json (hunk 1 above)
Comment (bnuluohf) [reply package.json:1]: @stacker-l 你自己写的登录？这个不行哈，改一下

**Pull request merged**